### PR TITLE
Fix keys menu: tokenterminal & shroom bug fix

### PR DESCRIPTION
**Pull request by montezdesousa**
# Description

* `tokenterminal` command not doing anything when called
* `shroom` command is always retrieving `not defined` on key setup

* Fix bugs, add keys to SDK and add unittests to recent changes 

Testing in terminal:
`keys` -> `tokenterminal -k test`
`keys` -> `shroom -k test`

Testing in SDK:
`keys.tokenterminal(key="test")`
`keys.shroom(key="test")`

### CODE REVIEW

**Review by hjoaquim — APPROVED** [review]
image

Awesome job, thanks for such attention to detail.

File: openbb_terminal/api.py [hunk 1]
```diff
@@ -1931,9 +1934,8 @@
     "keys.messari": {"model": "openbb_terminal.keys_model.set_messari_key"},
     "keys.eodhd": {"model": "openbb_terminal.keys_model.set_eodhd_key"},
     "keys.santiment": {"model": "openbb_terminal.keys_model.set_santiment_key"},
-    "keys.mykeys": {"model": "openbb_terminal.keys_model.get_keys"},
-    "keys.set_keys": {"model": "openbb_terminal.keys_model.set_keys"},
-    "keys.get_keys_info": {"model": "openbb_terminal.keys_model.get_keys_info"},
+    "keys.tokenterminal": {"model": "openbb_terminal.keys_model.set_tokenterminal_key"},
```
[the remark this answers is not in the document]
Comment (montezdesousa) [reply openbb_terminal/api.py:1937]: yup!

File: tests/openbb_terminal/cassettes/test_keys_controller/test_call_shroom[other1].yaml [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+      Accept-Encoding:
+      - gzip, deflate
+      Connection:
+      - keep-alive
+      Content-Length:
+      - '0'
+      User-Agent:
+      - python-requests/2.28.1
+      x-api-key:
+      - '1234'
```
[the remark this answers is not in the document]
Comment (montezdesousa) [reply tests/openbb_terminal/cassettes/test_keys_controller/test_call_shroom[other1].yaml:16]: U right it should, but as 1234 is actually a mock key itself don't think it's a problem.

File: openbb_terminal/keys_model.py [hunk 3]
```diff
@@ -37,6 +37,14 @@
 
 sys.tracebacklimit = 0
 
+# README PLEASE:
```
[the remark this answers is not in the document]
Comment (montezdesousa) [reply openbb_terminal/keys_model.py:40]: Imo the whole comment it's a bit file specific or you referring just to the api dictionary?

File: tests/openbb_terminal/cassettes/test_keys_model/test_set_shroom_key[args0-False-True-defined, test failed].yaml [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+      Accept-Encoding:
+      - gzip, deflate
+      Connection:
+      - keep-alive
+      Content-Length:
+      - '0'
+      User-Agent:
+      - python-requests/2.28.1
+      x-api-key:
+      - test_key
```
[the remark this answers is not in the document]
Comment (montezdesousa) [reply tests/openbb_terminal/cassettes/test_keys_model/test_set_shroom_key[args0-False-True-defined, test failed].yaml:16]: ^

File: tests/openbb_terminal/cassettes/test_keys_model/test_set_shroom_key[args1-False-False-defined, test failed].yaml [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+      Accept-Encoding:
+      - gzip, deflate
+      Connection:
+      - keep-alive
+      Content-Length:
+      - '0'
+      User-Agent:
+      - python-requests/2.28.1
+      x-api-key:
+      - test_key
```
[the remark this answers is not in the document]
Comment (montezdesousa) [reply tests/openbb_terminal/cassettes/test_keys_model/test_set_shroom_key[args1-False-False-defined, test failed].yaml:16]: ^

File: openbb_terminal/keys_model.py (hunk 3 above)
[the remark this answers is not in the document]
Comment (hjoaquim) [reply openbb_terminal/keys_model.py:40]: yap, to the api dict

[the remark this answers is not in the document]
Comment (montezdesousa) [reply openbb_terminal/keys_model.py:40]: We can make a referral to this https://github.com/OpenBB-finance/OpenBBTerminal/blob/main/CONTRIBUTING.md#add-api-endpoint ?

[the remark this answers is not in the document]
Comment (hjoaquim) [reply openbb_terminal/keys_model.py:40]: Yes, that would do

**Pull request merged**